.gitignore

```diff
@@ -20,3 +20,6 @@ Build
 *.swp
 *.orig
 .arcconfig
+
+*.tgz
```

Comment:
It's not clear what this is blocking here, and would prevent people adding .tgz examples in the future (for example). Can you explain why this is needed?

Reply:
Still need to know why this *.tgz is here. Can you remove this from the commit or justify why it's necessary?